targets/posix/nanoCLR/Target_HAL.cpp

```
@@ -0,0 +1,24 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include <iostream>
+#include <string>
+
+void HAL_Windows_Debug_Print(char *szText)
```

Clang (14.0.6)

[warning] pointer parameter 'szText' can be pointer to const

(readability-non-const-parameter)